### PR TITLE
Update ru_ru.json

### PR DESCRIPTION
**Pull request by Pahuhopu**
**-LINES ADDED AND TRANSLATED:**
  "block.farmersdelight.budding_tomatoes": "Budding Tomato Vine",
  "block.farmersdelight.sandy_shrub": "Sandy Shrub",
  "item.farmersdelight.rotten_tomato": "Rotten Tomato",
  "item.farmersdelight.glow_berry_custard": "Glow Berry Custard",
  "item.farmersdelight.salmon_roll": "Salmon Roll",
  "item.farmersdelight.cod_roll": "Cod Roll",
  "item.farmersdelight.kelp_roll": "Kelp Roll",
  "item.farmersdelight.kelp_roll_slice": "Kelp Roll Slice",
  "item.farmersdelight.bone_broth": "Bone Broth",
  "item.farmersdelight.mushroom_rice": "Mushroom Rice",
  "block.farmersdelight.rice_roll_medley_block": "Rice Roll Medley",
  "farmersdelight.container.recipe_book.cookable": "Showing Cookable",
  "farmersdelight.advancement.place_organic_compost": "Advanced Composting",
  "farmersdelight.advancement.place_organic_compost.desc": "Place down some Organic Compost. It composts better with sun, water and mushrooms!",
  "farmersdelight.advancement.get_fd_seed": "Crops of the Wild",
  "farmersdelight.advancement.get_fd_seed.desc": "Four new crops may be found in the wild, across many climates... or maybe in a chest somewhere.",
  "farmersdelight.advancement.harvest_ropelogged_tomato": "Tall-mato",
  "farmersdelight.advancement.harvest_ropelogged_tomato.desc": "Hang some rope above a tomato crop to make it grow taller",
  "farmersdelight.advancement.hit_raider_with_rotten_tomato": "Boo! Hiss!",
  "farmersdelight.advancement.hit_raider_with_rotten_tomato.desc": "Throw a Rotten Tomato at one of these pesky raiders!",
  "farmersdelight.subtitles.cabinet.open": "Cabinet opens",
  "farmersdelight.subtitles.cabinet.close": "Cabinet closes",
  "farmersdelight.subtitles.rotten_tomato.throw": "Rotten Tomato flies",
  "farmersdelight.subtitles.rotten_tomato.hit": "Rotten Tomato splatters",
  "death.attack.farmersdelight.stove": "%1$s was grilled to perfection",
  "death.attack.farmersdelight.stove.player": "%1$s was thrown on the grill by Chef %2$s",

**-LINES REMOVED:**
  "farmersdelight.advancement.place_stove": "Разожгу гриль",
  "farmersdelight.advancement.place_stove.desc": "Поставьте кухонную плиту",
  "farmersdelight.advancement.drink_hot_cocoa": "Чашка счастья",
  "farmersdelight.advancement.drink_hot_cocoa.desc": "Избавьтесь от негативных эффектов, выпив бутылочку с горячим какао",

**-LINES EDITED:**
 -before:
  "block.farmersdelight.tomatoes": "Томаты",
  "block.farmersdelight.wild_tomatoes": "Томатная лоза",
  "farmersdelight.subtitles.tomato.pick": "Лопание помидоров",
 -after:
  "block.farmersdelight.tomatoes": "Томатная лоза", //changed for more exact translation (like in en_us file)
  "block.farmersdelight.wild_tomatoes": "Куст томатов", //changed for more exact translation (like in en_us file)
  "farmersdelight.subtitles.tomato.pick": "Томат сорван", //'Лопание' is more like 'Burst' or 'Smash' so i`ve changed it to 'сорван' (it means 'Pick' or 'Harvest')

**-ALSO advancement lines had been put in order like in en_us file so this one have exact same number of lines (289)**

### CODE REVIEW

File: src/main/resources/assets/farmersdelight/lang/ru_ru.json [hunk 1]
```diff
@@ -115,6 +118,7 @@
   "item.farmersdelight.sweet_berry_cookie": "Печенье из сладких ягод",
   "item.farmersdelight.honey_cookie": "Медовое печенье",
   "item.farmersdelight.melon_popsicle": "Арбузное мороженое",
+  "item.farmersdelight.glow_berry_custard": "Крем из светящихя ягод",
```
Comment (DmitryScaletta) [inline src/main/resources/assets/farmersdelight/lang/ru_ru.json]: typo: светящихcя

Comment (Pahuhopu) [reply src/main/resources/assets/farmersdelight/lang/ru_ru.json]: > typo: светящихcя

спасибо! сейчас поправлю

File: src/main/resources/assets/farmersdelight/lang/ru_ru.json [hunk 2]
```diff
@@ -115,6 +118,7 @@
   "item.farmersdelight.sweet_berry_cookie": "Печенье из сладких ягод",
   "item.farmersdelight.honey_cookie": "Медовое печенье",
   "item.farmersdelight.melon_popsicle": "Арбузное мороженое",
+  "item.farmersdelight.glow_berry_custard": "Крем из светящихся ягод",
```
Comment (Pahuhopu) [inline src/main/resources/assets/farmersdelight/lang/ru_ru.json:121]: исправлено!

**Comment by vectorwing** [comment]
Thanks for the update (and the peer review, too)!

**Pull request merged**